release-1.8/release_notes_draft.md

```diff
@@ -170,8 +170,8 @@ across the system. Here's the release [scalability validation report].
 
 * Advanced auditing has graduated from `v1alpha1` to `v1beta1` with the
   following changes to the default behavior.
-  * Advanced auditing is enabled by default.
-  * The `--audit-policy-file` is now required unless the `AdvancedAudit`
+  * Advanced auditing is the default auditing mechanism, which changes the auditing behavior
+  * The `--audit-policy-file` is now required to enable auditing unless the `AdvancedAudit`
```

Comment:
How to enable auditing ? Is there a flag enabling auditing ?

Reply:
See https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#advanced-audit. Should we link to that here?

Reply:
Note, that it's been changed for 1.8 release. This page will soon look like this: https://github.com/kubernetes/kubernetes.github.io/blob/release-1.8/docs/tasks/debug-application-cluster/audit.md#advanced-audit

Reply:
> How to enable auditing ? Is there a flag enabling auditing ?
 
That is why the phrase "audit logging is not enabled by default" is confusing. It is enabled by default, it's just that it doesn't log anything unless you provide a policy. It is as if the default policy was empty

Reply:
I think that's an implementation detail. From the user's perspective audit logging isn't enabled if nothing is getting audit logged.